test/requests/workspace_symbol_test.rb

```diff
@@ -95,4 +95,17 @@ def test_finds_default_gem_symbols
     result = RubyLsp::Requests::WorkspaceSymbol.new("Pathname", @index).run
     refute_empty(result)
   end
+
+  def test_does_not_include_private_constants
+    @index.index_single(RubyIndexer::IndexablePath.new(nil, "/fake.rb"), <<~RUBY)
+      class Foo
+        CONSTANT = 1
+        private_constant(:CONSTANT)
+      end
+    RUBY
+
+    result = RubyLsp::Requests::WorkspaceSymbol.new("Foo::CONSTANT", @index).run
+    assert_equal(1, result.length)
```

Comment:
I may be misunderstanding the test, but if we're looking up `Foo::CONSTANT` then why is `Foo` in the results?

Reply:
Ah, I see. That may be worth a code comment.